src/aktualizr_repo/main.cc

```diff
@@ -24,7 +24,10 @@ int main(int argc, char **argv) {
     ("keyname", po::value<std::string>(), "name of key's role")
     ("repotype", po::value<std::string>(), "director|image")
     ("correlationid", po::value<std::string>()->default_value(""), "correlation id")
-    ("keytype", po::value<std::string>()->default_value("RSA2048"), "UPTANE key type");
+    ("keytype", po::value<std::string>()->default_value("RSA2048"), "UPTANE key type")
+    ("targetname", po::value<std::string>(), "target's name")
+    ("targethash", po::value<std::string>(), "target's hash")
+    ("targetlength", po::value<uint64_t>(), "target's length");
```

Comment:
These three new options should have some explanation here that they are intended for a special purpose. Something like "target name (for adding metadata without an actual file)".